cheerio-scraper/src/crawler_setup.js

```diff
@@ -201,7 +205,10 @@ class CrawlerSetup {
 
         if (this.evaledPrepareRequestFunction) {
             try {
-                await this.evaledPrepareRequestFunction({ request, Apify });
+                const { customData } = this.input;
+                const sessionId = session ? session.id : undefined;
+                const proxyInfo = this.proxyConfiguration.newProxyInfo(sessionId);
+                await this.evaledPrepareRequestFunction({ request, session, proxyInfo, Apify, customData });
```

Comment:
`prepareRequestFunction` gets `proxyInfo` as a parameter so no need to do this here. Just add it on line 188 and pass it here

Reply:
https://sdk.apify.com/docs/typedefs/prepare-request-inputs#docsNav

Reply:
Also means that all the above changes are not needed.

Reply:
Ahh, I see. Changed.